backend/dataall/modules/dataset_sharing/api/resolvers.py

```diff
@@ -191,6 +191,7 @@ def resolve_consumption_data(context: Context, source: ShareObject, **kwargs):
             return {
                 's3AccessPointName': S3AccessPointName,
                 'sharedGlueDatabase': (ds.GlueDatabaseName + '_shared_' + source.shareUri)[:254] if ds else 'Not created',
+                's3bucketName': ds.S3BucketName if ds else 'NotFound',
```

Comment:
`if ds` is already checked at line 186, if we want to return some default values it should be in an else branch

Reply:
I agree. Removed the else portion.